app/src/main/java/uk/gov/android/securestore/SharedPrefsStore.kt

```diff
@@ -57,6 +63,7 @@ class SharedPrefsStore(
                         authPromptConfig
                     )
                 } ?: continuation.resumeWith(Result.success(null))
+                authenticator.close(context)
```

Comment:
could you also move this line to a finally block please?

Reply:
Done, thank you